sid/matching_probabilities.py

```diff
@@ -68,7 +68,7 @@ def _get_transition_matrix_from_params(params, states, variable, model_name):
             own_prob.index = own_prob.index.get_level_values("name")
             trans_mat = _create_transition_matrix_from_own_prob(own_prob)
         else:
-            trans_mat = params.loc[loc, "value"].unstack()
+            trans_mat = params.loc[loc]["value"].unstack()
```

Comment:
I don't understand why this was necessary. Maybe make a comment?

Reply:
It's not necessary but saves a lot of the pandas lexsort PerformanceWarnings. I added a comment.